lib/reversetunnel/agent.go

```diff
@@ -361,6 +357,7 @@ func (a *Agent) startHeartbeat(conn ssh.Conn) {
 			errC <- err
 			return
 		}
+		hb.SendRequest("ping", false, nil)
```

Comment:
this speeds up integration tests because it shoots the first ping without waiting